app/controllers/idv/capture_doc_controller.rb

```diff
@@ -52,5 +61,9 @@ def process_result(result)
     def token
       params[:token]
     end
+
+    def session_uuid
+      params[:session]
```

Comment:
can we call the param `session_id` or `session_uuid`?

Reply:
> can we call the param `session_id` or `session_uuid`?

I'd originally named it `session_uuid`, but then opted for this because:

- Consistency with how the `request_token` field is referenced by the single-word `token` parameter
   - Possibly related: Human-friendly parameter names
- Length, as noted below in your comment at https://github.com/18F/identity-idp/pull/4153#discussion_r480215221 and in my original comment "Open question"

That being said, I do think the inconsistency makes it harder to juggle. Also, I'd run into at least one occurrence where naming it `session` appeared to conflict with the global session variable.

Reply:
I don't think consistency of the super ambiguous `token` is a good thing to match to LOL

Reply:
One extra thing which is probably relevant here:

https://github.com/18F/identity-idp/blob/6fdf921dbe1268d71a63653ed430dcc270f56412/spec/features/idv/doc_auth/send_link_step_spec.rb#L33-L34

So at least we'd probably need to use a hyphen instead of an underscore.

This gotcha also helps explain this route duplication, which was previously confusing to me:

https://github.com/18F/identity-idp/blob/6fdf921dbe1268d71a63653ed430dcc270f56412/config/routes.rb#L266-L267

Given that we're now hyphenating `document_capture`:

https://github.com/18F/identity-idp/blob/0ae29b253facffc039b6b3ab7bbea7896b995261/app/services/idv/steps/send_link_step.rb#L31

I assume we'll also need to either add or replace with a new `document-capture` to the Acuant SDK routes above.

Reply:
Maybe we should just make the URL in the SMS a very short one and redirect from there our normal ones that use underscores?

`/handoff?session=&token=`

Reply:
> Maybe we should just make the URL in the SMS a very short one and redirect from there our normal ones that use underscores?
> 
> `/handoff?session=&token=`

I like the idea. It might also complement well what we'll need to do with https://github.com/18F/identity-idp/pull/4153#pullrequestreview-478826227, where toggling steps as complete based on enabled might also be appropriate to handle in this sort of "handoff" redirect.

While it's not super short, I wonder if a decent compromise would be:

```rb
get '/verify/capture-doc' => 'capture_doc#index'
```

Replacing this:

https://github.com/18F/identity-idp/blob/6fdf921dbe1268d71a63653ed430dcc270f56412/config/routes.rb#L317

Where `index` could act as this handoff / step completion toggling / redirect route.

I guess we could still go shorter, and redirect to the same `capture_doc` controller? The downside being that we're not really consistent with route -> controller mapping, though arguably that's already the case with `capture-doc`, and at least the inconsistency would be a little more explicit.

Example:

```rb
    scope '/verify', module: 'idv', as: 'idv' do
      get '/' => 'capture_doc#handoff'
      # ...
```

Reply:
In rebased b5f1334ea68533771810bbf9b83022da279d3482:

- Parameter is renamed from `session` to `document-capture-session`
- We only send one or the other of `token` or `document-capture-session`, depending on feature flags
- The index route `/verify/capture-doc` serves as a handoff route